sw/device/silicon_creator/lib/boot_svc/boot_svc_empty.h

```diff
@@ -0,0 +1,59 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef OPENTITAN_SW_DEVICE_SILICON_CREATOR_LIB_BOOT_SVC_BOOT_SVC_EMPTY_H_
+#define OPENTITAN_SW_DEVICE_SILICON_CREATOR_LIB_BOOT_SVC_BOOT_SVC_EMPTY_H_
+
+#include <stdint.h>
+
+#include "sw/device/lib/base/macros.h"
+#include "sw/device/silicon_creator/lib/boot_svc/boot_svc_header.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif  // __cplusplus
+
+enum {
+  kBootSvcEmptyType = 0xb4594546,
+  kBootSvcEmptyPayloadWordCount =
+      CHIP_BOOT_SVC_MSG_PAYLOAD_SIZE_MAX / sizeof(uint32_t),
+};
+
+/**
+ * An empty boot services message.
+ *
+ * This message type consists of a random payload that is as large as the
+ * largest boot services message. ROM_EXT can use an object of this type to
+ * initialize the boot services area in the retention SRAM or to clear a boot
+ * services request before writing the response. Silicon owner code can use an
```

Comment:
I would suggest replacing "can use" for "should use" because this will potentially be a documentation for whom will write the owner code.